Closure_Project/rest_api/migrations/0001_initial.py

```diff
@@ -0,0 +1,133 @@
+# Generated by Django 3.2.4 on 2021-06-27 14:11
```

Comment:
Are migration files safe to commit? (bug-wise, not security-wise)

Reply:
Generally yes, migration files are like any other code, at any point in time the migrations should match up to the models  (such that if you clone a repository, doing `makemirations rest_api` should not create any new files, in fact maybe I'll even make a github action for this)

Reply:
Perfect, thanks